### ISSUE TITLE
Sync integrated web-wallet login and extension

### ISSUE THREAD

**Issue by mradkov** [aeternity/superhero-ui#606]
The idea is if you have the extension installed - to be still presented the section with the web-wallet, synced with your account credentials.

**Comment by davidyuk** [comment]
I don't think that we need new designs for that. This task is technically complex, but it should be doable. Most probable it required some Vuex refactoring, setting a syncing of state in different directions.
